epictrack-web/src/components/settings/tabs/UserManagementList.tsx

```diff
@@ -0,0 +1,414 @@
+import { MRT_ColumnDef, MRT_TableOptions } from "material-react-table";
+import { Box, Grid, IconButton } from "@mui/material";
+import { getErrorMessage } from "utils/axiosUtils";
+import { useCachedState } from "hooks/useCachedFilters";
+import Icons from "components/icons";
+import { IconProps } from "components/icons/type";
+import { ColumnFilter } from "components/shared/MasterTrackTable/type";
+import { ETChip } from "components/shared/chip/ETChip";
+import { ETParagraph } from "components/shared";
+import { getSelectFilterOptions } from "components/shared/MasterTrackTable/utils";
+import { Restricted } from "components/shared/restricted";
+import { searchFilter } from "components/shared/MasterTrackTable/filters";
+import { showNotification } from "components/shared/notificationProvider";
+import MasterTrackTable from "components/shared/MasterTrackTable";
+import TableFilter from "components/shared/filterSelect/TableFilter";
+import TrackSelect from "components/shared/TrackSelect";
+import elevatedRoleService from "services/elevatedRoleService";
+import staffElevatedRoleService from "services/staffElevatedRoleService/staffElevatedRoleService";
+import staffService from "services/staffService/staffService";
+import { ROLES } from "constants/application-constant";
+import { ElevatedRole } from "models/elevated_role";
+import { StaffElevatedRole, StaffWithElevatedRoles } from "models/staff";
+import { Palette } from "styles/theme";
+
+const EditIcon: FC<IconProps> = Icons["PencilEditIcon"];
+const CheckIcon: FC<IconProps> = Icons["CheckIcon"];
+const CancelIcon: FC<IconProps> = Icons["CloseXIcon"];
+
+const userManagementListColumnFiltersCacheKey =
+  "user-management-listing-column-filters";
+
+const UserManagementList = () => {
+  const [columnFilters, setColumnFilters] = useCachedState<ColumnFilter[]>(
+    userManagementListColumnFiltersCacheKey,
+    [
+      {
+        id: "position.name",
+        value: ["IPE"],
+      },
+      {
+        id: "is_active",
+        value: [true],
+      },
+    ]
+  );
+  const [elevatedRoles, setElevatedRoles] = useState<ElevatedRole[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [staffElevatedRoles, setStaffElevatedRoles] = useState<
+    StaffElevatedRole[]
+  >([]);
+  const [staffs, setStaffs] = useState<StaffWithElevatedRoles[]>([]);
+
+  const fetchStaffs = async () => {
+    setLoading(true);
+    try {
+      const response = await staffService.getAll();
+      setStaffs((response.data as StaffWithElevatedRoles[]) || []);
+      setLoading(false);
+    } catch (error) {
+      showNotification("Could not load Staffs", { type: "error" });
+    }
+  };
+
+  const fetchStaffElevatedRoles = async () => {
+    setLoading(true);
+    try {
+      const response = await staffElevatedRoleService.getAll();
+      setStaffElevatedRoles((response.data as StaffElevatedRole[]) || []);
+      setLoading(false);
+    } catch (error) {
```

Comment:
Should this `setLoading(false)` since at this point we know that the loading isn't gonna happen?

Reply:
Fixed in [a672afa](https://github.com/bcgov/EPIC.track/pull/2552/commits/a672afa2b9909f75f6483273913a511cd99f590d)